src/net/sourceforge/kolmafia/request/MallPurchaseRequest.java

```diff
@@ -157,6 +179,17 @@ public long getAvailableMeat() {
 
   @Override
   public String color() {
+    if (getForbiddenStores().contains(String.valueOf(this.shopId))) {
+      // Try get the color from look and feel
+      Color color = UIManager.getColor("InternalFrame.closePressedBackground");
```

Comment:
Suspicious that suspicious that you've had to take an intended background colour but if it works it works. You could even take the colour that we use for the error state on the compactsidepane

Reply:
A bright cherry red sucks :P

Reply:
Hmm. Let's merge this but I think this is a minor bad decision